src/screens/pos-actions/pos-basic-action/pos-basic-action.tsx

```diff
@@ -271,14 +271,24 @@ export class PosBasicActionComponent extends React.Component<
 
     public addAmount(value: string) {
         const amount = value.replace(/,/g, '.');
+        const blockchainInstance = getBlockchain(this.props.blockchain);
+
+        const tokenConfig = getTokenConfig(this.props.blockchain, this.props.token.symbol);
+
+        const activeBalance = blockchainInstance.account
+            .amountFromStd(
+                new BigNumber(this.props.validators[0].amountDelegated.active),
+                tokenConfig.decimals
+            )
+            .toFixed();
         this.setState({ amount }, () => {
             const { insufficientFunds, insufficientFundsFees } = availableFunds(
                 amount,
                 this.props.account,
                 this.props.token,
                 this.props.chainId,
                 this.state.feeOptions,
-                this.props.validators[0].totalVotes
+                activeBalance
```

Comment:
Doesnt look good here!!!
Basic action screen is used for Unstake and Restake as well.... if you use availableBalance and not what the balance is used ... then its a problem

Reply:
Nu merge pe STAKE, merge doar pe Unstake si Claim Reward